src/ui/OGMessageItemBody/index.scss

```diff
@@ -4,9 +4,9 @@
   position: relative;
   display: inline-block;
   // TODO: Change fixing width and height lengths
-  width: 100%;
+  width: 320px;
```

Comment:
using this could result in overflow in mobile devices is this ok?

Reply:
We already have set the minWidth to 320px, so I think it won't be wrong for now